core/src/main/java/org/kiwix/kiwixmobile/core/settings/SliderPreference.java

```diff
@@ -168,12 +174,22 @@ protected void onBindView(View view) {
   private void setGrayState(View view) {
     SharedPreferenceUtil sharedPreferenceUtil = new SharedPreferenceUtil(getContext());
     boolean enabled = sharedPreferenceUtil.getPrefZoomEnabled();
+    NightModeConfig nightMode = new NightModeConfig(sharedPreferenceUtil,getContext());
+    boolean nightMode_enabled = nightMode.isNightModeActive();
 
     TextView titleView = view.findViewById(android.R.id.title);
     TextView summaryTV = view.findViewById(android.R.id.summary);
     if (!enabled) {
       titleView.setTextColor(Color.GRAY);
       summaryTV.setTextColor(Color.GRAY);
+    }else{
+      if(nightMode_enabled){
```

Comment:
There should be spaces before `(` & after `)`.
Check out our code style guidelines in the README of this repo.

Reply:
you can press `ctrl+alt+l` to format, our codestyle is part of this project. I generally recommend setting up a macro to format,optimise imports and save bound to ctrl+s

Reply:
I'll fix the spaces and use proper codestyle wherever required.